inventory.py

```diff
@@ -0,0 +1,61 @@
+from typing import List, Optional
+
+from item import ALL_ITEM_CLASSES
+
+
+class Passive:
+    """Class to define item (or more?) passive"""
+
+    def __init__(self, name: str, stats: dict, unique: bool = False):
+        self.name = name
+        self.unique = unique
+        self.stats = stats
+
+
+class Inventory:
+    """
+    Inventory class that stores the 6 items.
+    - From a list of Item names, create and store all items in side the attribute items.
+    - Add item passives to items stats if available. Handle unique passives.
+    """
+
+    def __init__(self, item_names: Optional[List[str]] = None):
```

Comment:
Why not get a list of item objects instead? I think it would make more sense to initialize items before putting them in the inventory, instead of having the inventory do that.

Reply:
It was setup this way make it easier for the user to add an item (using just a string instead of creating the item). 
But I see your point
Would you create the inventory outside of BaseChamp too ?

Reply:
No, let's define the inventory outside then !

Reply:
I misread the question! I meant I would create the inventory inside the champion directly, instead of a separate class.